ffs/scheduler/internal/sjstore/sjstore.go

```diff
@@ -87,29 +98,17 @@ func (s *Store) Finalize(jid ffs.JobID, st ffs.JobStatus, jobError error, dealEr
 func (s *Store) Dequeue() (*ffs.StorageJob, error) {
```

Comment:
Now we keep a `queued` slice as a cache of all pending Jobs. 
This slice is ordered by `CreatedAt` field, so dequeuing is always easy and also avoids querying Datastore.